torch/csrc/jit/codegen/cuda/predicate_compute.cpp

```diff
@@ -436,6 +436,7 @@ void UnswitchPredicate::predicateOn(Expr* tv_expr) {
 
   const auto gpu_lower = GpuLower::current();
   if (gpu_lower->predicateElimination().canOmitPredicate(tv_expr)) {
+    addParallelizedDomainPredicates(tv_expr);
```

Comment:
This change is to add predicates for parallel domains (e.g., `threadIdx.x < N`), even when tensors do not need predicates.